### DISCUSSION TITLE
Meeting of Remix CC (2023-10-08)

### DISCUSSION THREAD

**Discussion by drafish** [PlanckerLabs/remix-cc#84] [Meeting of Remix CC]
### 三季度工作成果
**开发计划**

1. 完成 Remix IDE 国际化的收尾工作
2. 完成 Remix 官网的国际化
3. 完成 Remix IDE镜像站 0.34.0 ~ 0.36.0 的版本升级
4. 除国际化以外，还向 Remix 提交了4个PR，3个merge，剩下一个未接受
5. 给 Remix 提了4个 issue，其中包含一些反馈和建议

**翻译计划**

1. 完成三季度新增的翻译工作
2. 帮助 Remix 团队完善了文档翻译与构建流程

**运营计划**

1. 产出6个教学视频
2. 翻译两个教学视频
3. 翻译 0.34.0 ~ 0.36.0 的更新日志
4. 产出三篇文档，Remix 国际化、中文镜像站、Remix 部署
5. 在 PET 做了一次分享 —— Remix 国际化

**研究计划**

1. 初步完成社区版 Remix VSCode 调研，产出一份开发计划

**对外合作**

1. 与星火链网达成合作，四季度会帮他们开发一个 Remix 插件
2. 与 FootPrint 就新版 OneClickDapp 达成合作意向，四季度继续推进

**未完成预期**

1. 社区版 Remix VSCode 开发计划不够详细，不足以向 GCC 发起 grants 申请，也没有到 PET 做技术分享。四季度继续寻找合适的主力开发，找到后共同制定进一步的开发计划，并且向 GCC 申请 grants 。
2. 与 Remix 团队重新讨论了下 grants 范围，但没有确定下来，四季度继续推进。

### 四季度工作规划

**开发计划**

1. Remix CC 官网开发（yiwen）
2. 镜像站维护
3. DGit 插件优化（待定）
4. Remix IDE 国际化功能维护（待定）
5. LearnETH 插件维护（待定）
6. 非 EVM 兼容链插件模板（待定）

**翻译计划**

1. 翻译四季度新增文档（yiwen）
2. 维护 Crowdin + Readthedocs（待定）

**运营计划** 

1. 产出6个教学视频
2. 翻译 Remix 官方的教学视频，加中文字幕（yiwen）
3. 财务公示 + 捐赠墙
4. twitter space 采访

**研究计划**

1. 与 Remix 团队讨论社区版 Remix VSCode，继续寻找主力开发，制定详细的开发计划，并且向 GCC 申请 grants ，启动开发计划

**对外合作**

1. 为星火链开发一个 Remix 插件
2. 与 FootPrint 联合开发新版 OneClickDapp

**Comment by drafish** [comment]
### DJQ (2023-10-01)
1. 教学视频制作 -- 在 Remix 上加载已部署的合约
2. 修复 Remix  官网国际化 PR 的冲突
3. remix ide 镜像站更新到了 v0.36.0 版本
4. 帮 mocca 和 yiwen review 教学视频
### DJQ (2023-10-08)
1. 三季度工作成果，四季度工作规划

**Comment by drafish** [comment]
### Mocca (2023-10-01)
1. 教学视频制作 -- remix analyzers

**Comment by yiwen4** [comment]
### yiwen (2023-10-01)
1. 教学视频完成制作 -- 部署 Remix IDE
2. 修改完善 -- remix analyzers教学视频 
3. 翻译 v0.36.0 版本更新日志 并发布到公众号

**Comment by yiwen4** [comment]
### yiwen (2023-10-08)
1. 完成9月翻译
2. 自媒体运营：注册gmail邮箱、注册b站，并发布8个视频、注册登链，并发布v0.36.0更新日志、上传视频到头条
3. 制定下季度运营计划
